lib/elastic_apm/config.rb

```diff
@@ -185,6 +186,18 @@ def capture_body=(value)
     end
     # rubocop:enable Metrics/MethodLength
 
+    # DEPRECATED
+    def disabled_spies=(list)
+      warn 'The option disabled_spies has been renamed to ' \
+        'disabled_instrumentations'
+      self.disabled_instrumentations = list
+    end
+
+    def enabled_spies
+      warn 'enabled_spies has been renamed to enabled_instrumentations'
+      enabled_instrumentations
+    end
```

Comment:
Is there a reason you didn't use the `deprecate` helper?

Reply:
I guess not, no 👍